app/waf/src/models/as3.model.ts

```diff
@@ -372,7 +372,9 @@ export class AS3WAFPolicy extends AS3Object {
     super();
 
     this.class = 'WAF_Policy';
-    this.name = policy.name;
+    if (policy.name) {
```

Comment:
Just wondering what's the else case..

AS3WAFPolicy's name is declared as `string`, is it possible to be `undefined`?

If it's OK, can we use `this.name = <string>policy.name;` instead of an `if`?

Reply:
It will be removed by our new as3 translation code in next commit. Needn't to improve it any more.